src/Lecture3.hs

```diff
@@ -226,11 +263,18 @@ types that can have such an instance.
   case). But comments still mention all types to avoid spoilers ;)
 -}
 
--- instance Functor Weekday where
--- instance Functor Gold where
--- instance Functor Reward where
--- instance Functor List1 where
--- instance Functor Treasure where
+-- instance Functor Weekday where -- Kind is `*`
+-- instance Functor Gold where -- Kind is `*`
+-- instance Functor Reward where -- Kind is `*`
+instance Functor List1 where
+  fmap :: (a -> b) -> List1 a -> List1 b
+  fmap f (List1 a []) = List1 (f a) []
+  fmap f (List1 a (a':as')) = List1 (f a) [] <> fmap f (List1 a' as')
```

Comment:
Same here that you don't need to pattern match on a list. And you also can use the `List1` constructor directly instead of `<>` 🙂 
```suggestion
  fmap f (List1 a as) = List1 (f a) (map f as)
```